bucket/pipp.json

```diff
@@ -0,0 +1,41 @@
+{
+    "description": "An Application designed for pre-processing planetary images before stacking them with image stacking software such as Registax.",
+    "homepage": "https://sites.google.com/site/astropipp/home",
+    "version": "2.5.9",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://gist.githubusercontent.com/MCOfficer/f308c0b562272fa5b4c6bb3163c72509/raw/8c06bac589c8e06a8278e46ea196bab7a9d742d2/pipp-2.5.9-license.txt"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://sites.google.com/site/astropipp/downloads/pipp_install_x64_2.5.9.zip",
+            "hash": "231f08bf36aa812946158d6239861bd9a3a328d33124afd357a35dc28d623efe"
+        },
+        "32bit": {
+            "url": "https://sites.google.com/site/astropipp/downloads/pipp_install_x86_2.5.9.zip",
+            "hash": "87dfaa846e75e265a004961aa785b5544ec7cf8ae0cf4c7443a9c677172bba0d"
+        }
+    },
+    "innosetup": true,
```

Comment:
As i just noticed, now that innosetup is fixed, [this is bullshit](https://github.com/lukesampson/scoop/issues/3339#issuecomment-487362746)

Which poses the question: can i extract the archive and *then* run innosetup?
and can i run innosetup on a file with version-specific name? autoupdate doesn't seem to set any values for `installer`.

Reply:
It is not innosetup at all.

![A](https://i.imgur.com/bYLl8fE.png)